src/Sarif.Validation/Sarif.Validation.csproj

```diff
@@ -27,14 +27,18 @@
       <HintPath>..\packages\Microsoft.CodeAnalysis.SarifValidationLibraries.1.0.0\lib\net46\Microsoft.JSON.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Json.Schema.ToDotNet, Version=0.8.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Json.Schema.ToDotNet.0.8.0\lib\net45\Microsoft.Json.Schema.ToDotNet.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
I'm surprised this project didn't already have a reference to Json.Schema.ToDotNet.